packages/corelight/kibana/dashboard/corelight-0cfc8a95-4979-4265-b9ad-91edf63f01a9.json

```
@@ -0,0 +1,1957 @@
+                                        "type": "combined"
+                                    },
+                                    "query": {}
+                                },
+                                {
+                                    "$state": {
+                                        "store": "appState"
+                                    },
+                                    "meta": {
+                                        "alias": null,
+                                        "disabled": false,
+                                        "field": "file.mime_type",
+                                        "index": "9c51268e-315a-4270-aab0-c370f3461b91",
+                                        "key": "file.mime_type",
+                                        "negate": false,
+                                        "type": "exists"
+                                    },
+                                    "query": {
+                                        "exists": {
+                                            "field": "file.mime_type"
+                                        }
+                                    }
+                                }
+                            ],
+                            "internalReferences": [],
+                            "query": {
+                                "language": "kuery",
+                                "query": ""
+                            },
+                            "visualization": {
+                                "columns": [
+                                    {
+                                        "columnId": "41bd222e-daaf-4170-865a-32211359d183",
+                                        "isMetric": false,
+                                        "isTransposed": false
+                                    },
+                                    {
+                                        "columnId": "b541bdcb-e210-4ae1-8ecf-9b61254c6a0d",
+                                        "isMetric": true,
+                                        "isTransposed": false
+                                    }
+                                ],
+                                "layerId": "aa62b6fc-5c2a-4906-949c-6f312a971d54",
+                                "layerType": "data"
+                            }
+                        },
+                        "title": "",
+                        "type": "lens",
+                        "visualizationType": "lnsDatatable"
+                    },
+                    "enhancements": {}
+                },
+                "gridData": {
+                    "h": 16,
+                    "i": "7071f201-af3a-4a3d-a4b7-f4ca911951dc",
+                    "w": 24,
+                    "x": 0,
+                    "y": 40
+                },
+                "panelIndex": "7071f201-af3a-4a3d-a4b7-f4ca911951dc",
+                "title": "Top Transmitting (tx_host) Hosts - # Files [Logs Corelight]",
```

```suggestion
                "title": "Top Transmitting (tx_host) Hosts by number of files [Logs Corelight]",
```
Can we replace `#` with `number` or `number of`. 
Same in other places too.